### DISCUSSION TITLE
Can't swipe left on release notifications

### DISCUSSION THREAD

**Discussion by neorth** [community/community#91816] [Mobile]
Starting yesterday I can no longer swipe left on release notifications in the Android app.

https://github.com/community/community/assets/8577089/277d2b48-995c-45fa-9293-63d9b897ff91

Edit: The version I'm running is v1.146.0-beta (10172)

**Comment by alcere** [comment]
Hi @neorth, we are sorry for the troubles here! We have fixed the issue, please stay tuned for updates tomorrow, thanks for reporting!
